packages/fxa-content-server/app/styles/modules/_settings-totp.scss

```diff
@@ -150,8 +150,24 @@
 }
 
 .save-option {
+  background-color: transparent;
+  color: $link-color-default;
+  display: inline-block;
+  font-weight: 400;
+  height: initial;
+  margin: -10px 0;
+  padding: 10px 0;
```

Comment:
Instead of adding a negative margin here, let's reduce some padding elsewhere. If we reduce `.recovery-key-container` margin-bottom to `15px` instead of `25px`, and then change the margin of`.save-options` to `15px 0 10px`, we'll get the same affect.